ansible/roles/iroha-docker/templates/config.docker.j2

```diff
@@ -1,4 +1,12 @@
 {
+{% if iroha_log_debug %}
+  "log": {
+      "level": "debug",
+      "patterns": {
+        "trace": "[%Y-%m-%d %H:%M:%S.%F][th:%t][%l] %n %v"
```

Comment:
we need to check that this pattern will be supported by our elk https://soramitsu.atlassian.net/browse/DOPS-557